core/spec/models/spree/store_spec.rb

```diff
@@ -37,19 +37,32 @@
   end
 
   describe ".default" do
-    let!(:store)    { create(:store) }
-    let!(:store_2)  { create(:store, default: true) }
+    before do
+      Spree::Store.delete_all
```

Comment:
> I still have no idea how a bunch of other tests (especially controller/feature tests) are managing to pass without any apparent creation of a spree default store

@jrochkind are there any in particular that you see that look weird? I was curious about this also so I took a look at a few specs and the ones I looked at all seemed to be creating a default store in one way or another:

- [frontend/spec/features/checkout_spec.rb](https://github.com/solidusio/solidus/blob/3a8592d6861f12a9edd063858652729facfde377/frontend/spec/features/checkout_spec.rb#L4) (via ["checkout setup"](https://github.com/solidusio/solidus/blob/3a8592d6861f12a9edd063858652729facfde377/frontend/spec/support/shared_contexts/checkout_setup.rb))
- [frontend/spec/features/order_spec.rb](https://github.com/solidusio/solidus/blob/3a8592d6861f12a9edd063858652729facfde377/frontend/spec/features/order_spec.rb#L4) (via [OrderWalkthrough](https://github.com/solidusio/solidus/blob/3a8592d6861f12a9edd063858652729facfde377/core/lib/spree/testing_support/order_walkthrough.rb#L23))
- [api/spec/controllers/spree/api/orders_controller_spec.rb](https://github.com/solidusio/solidus/blob/3a8592d6861f12a9edd063858652729facfde377/api/spec/controllers/spree/api/orders_controller_spec.rb#L7) (via [order factory](https://github.com/solidusio/solidus/blob/3a8592d6861f12a9edd063858652729facfde377/core/lib/spree/testing_support/factories/order_factory.rb#L15))
- [backend/spec/controllers/spree/admin/orders_controller_spec.rb](https://github.com/solidusio/solidus/blob/3a8592d6861f12a9edd063858652729facfde377/backend/spec/controllers/spree/admin/orders_controller_spec.rb#L6)